packages/x-charts/src/internals/plugins/featurePlugins/useChartInteraction/useChartInteraction.ts

```diff
@@ -45,7 +41,7 @@ export const useChartInteraction: ChartPlugin<UseChartInteractionSignature> = ({
   });
 
   const setItemInteraction = useEventCallback(function setItemInteraction(
-    newItem: ChartItemIdentifierWithData<ChartSeriesType>,
+    newItem: ChartItemIdentifier<ChartSeriesType>,
```

Comment:
@JCQuintas This might be buggy for sankey, because the `setInteraction(newItem)` can be called with an item including the additional properties which will be saved in the sotre.

I'm considering adding in seriesConfig a method to remove addition properties `simplifyIdentifier: (item: ChartItemIdentifierWithData) => identifier: ChartItemIdentifier`

```ts
setItemInteraction(newItem:
  ChartItemIdentifier<ChartSeriesType>
  | ChartItemIdentifierWithData<ChartSeriesType>
) {
  const identifier = seriesConfig[identifier.type].simplifyIdentifier?.(newItem) ?? newItem;
  if (!fastObjectShallowCompare(store.state.interaction.item, identifier)) {
    // ....
  }
}